yaml/com/ibm/VPD/FRUManager.interface.yaml

```diff
@@ -0,0 +1,125 @@
+description: >
+    Implement to manage VPD data in system.
+
+methods:
+    - name: WriteKeyword
+      description: >
+          Method to update the value of a keyword with the given value. Updates
+          keyword on both cache and hardware
+      parameters:
+          - name: path
+            type: string
+            description: >
+                Inventory Path of the FRU whose keyword needs to be updated.
+          - name: data
+            type:
+                variant[struct[string,string,array[byte]],
+                struct[string,array[byte]]]
+            description: >
+                Contains data required to write either IPZ type VPD or Keyword
```

Comment:
"Contains the data required to update of type either IPZ or keyword format" ?

Reply:
No, it can be of any type
Updated the comment